File: VB/MyXtraGrid/Custom Grid Control/ColumnCustomizationListBoxEx.vb
Imports DevExpress.XtraGrid.Views.Grid.Customization
Imports System
Imports System.Collections.Generic
Imports System.Linq

Namespace MyXtraGrid
    Public Class ColumnCustomizationListBoxEx
        Inherits ColumnCustomizationListBox

        Public Sub New(ByVal form As CustomizationForm)
            MyBase.New(form)
        End Sub
        Public Overrides Function GetItemHeight() As Integer
            Dim view As MyGridView = TryCast(Me.View, MyGridView)
            If view IsNot Nothing Then
                Return view.CustomColumnRowHeight
            End If
            Return MyBase.GetItemHeight()
        End Function
    End Class
End Namespace

File: CS/MyXtraGrid/Custom Grid Control/ColumnCustomizationListBoxEx.cs
using DevExpress.XtraGrid.Views.Grid.Customization;
using System;
using System.Collections.Generic;
using System.Linq;

namespace MyXtraGrid {
    public class ColumnCustomizationListBoxEx : ColumnCustomizationListBox {
        public ColumnCustomizationListBoxEx(CustomizationForm form)
            : base(form) {
        }
        public override int GetItemHeight() {
            MyGridView view = this.View as MyGridView;
            if(view != null)
                return view.CustomColumnRowHeight;
            return base.GetItemHeight();
        }
    }
}
